### DISCUSSION TITLE
config/intro/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/vuepress-theme-plume#123] [docs-comment]
# config/intro/

VuePress 配置文件 概述 VuePress 站点的基本配置文件是 .vuepress/config.js ，但也同样支持 TypeScript 配置文件。 你可以使用 .vuepress/config.ts 来得到更好的类型提示。 具体而言，VuePress 对于配置文件的路径有着约定（按照优先顺序）： 当前工作目录 cwd 下： vuepre...

https://plume.pengzhanbo.cn/config/intro/

**Comment by ljporljp** [comment]
好看

**Comment by xlcaptain** [comment]
import { defineThemeConfig } from 'vuepress-theme-plume'
import navbar from './navbar'

export default defineThemeConfig({
  // 在这里配置主题
  profile: {
    name: 'Your name',
  },
  navbar,
}) 

配置了，config.ts怎么获取你是不是要写清楚啊？默认安装plume.config.ts和congfig.ts都有，但是又没有使用plume.config.ts。这不是既要又要嘛

**Reply by pengzhanbo** [reply]
我认为这个文档已经表述得很清楚了，`plugin.config.ts` 就是作为 `config.ts` 在配置主题时的补充和优化。
而且这也不是既要又要，我不会默认用户都有一定的技术水平，用户把主题配置写在哪个文件都能够正常使用主题，主题内部也会对配置进行合并。而说明文档默认使用 `config.ts` 对于大部分用户足够友好，对于有些技术底子的用户而言，这份文档也足够理解并选择哪种方式是更为适用。

**Comment by 929721152** [comment]
测试
